M2/Macaulay2/tests/normal/hashtables.m2

```diff
@@ -24,32 +26,6 @@ assert( class(tally{1,2,3}+set{3,4,5}) === Tally )
 assert( mutable( merge(new Type of List, new Type of Ring, first) ) )
 assert( class((new VirtualTally from {1 => -1, 2 => -2, 3 => -3}) + tally {1,2,3}) === VirtualTally )
 
-end
```

Comment:
Just curious, are we going towards not using `generateAssertions` anymore? Removing the bottom part makes it so you have to generate the assertions above by hand.

Reply:
didn't know `generateAssertions` even existed. that being said, I thought we switched to assertions being of the form `Equation { lhs, rhs }` ?

Reply:
The point of `generateAssertions` is to not have to write the `rhs` manually.

Reply:
I understand that. mine was that it doesn't generate assertions of the form `Equation{...}`.

Reply:
Ah, yes. `generateAssertions` is way older, definitely before `Equation { lhs, rhs }` was a thing.

Reply:
I don't see the point of keeping the `generateAssertions` code in the file.  Once you've generated the assertions and have pasted the desired calls to `assert` for your tests, then it's just cruft.